### ISSUE TITLE
Cipher supports in cryptodev-linux

### ISSUE THREAD

**Issue by Ashok-d-attikatti** [cryptodev-linux/cryptodev-linux#83]
Hi,

which cipher modes are supported by cryptodev-linux ? 
hope AES-ecb,cbc,ofb,cfb & ctr modes are supported. kindly confirm.

Thanks,
Ashoka A D

**Comment by cristian-stoica** [comment]
Does https://github.com/cryptodev-linux/cryptodev-linux/blob/master/crypto/cryptodev.h answer your question?

**Comment by Ashok-d-attikatti** [comment]
Yes, may I know from the OpenSSL can access the ofb and cfb modes.?
